pytest_qaseio/storage.py

```diff
@@ -1,15 +1,31 @@
 import io
-from typing import Protocol
+from typing import Protocol, cast
 
 import qaseio
-import qaseio.apis
+from qaseio.configuration import Configuration
```

Comment:
```suggestion
from qaseio import configuration as qaseio_config
```
To be consistent with `api_client.py`